package.json

```diff
@@ -10,7 +10,6 @@
 		"com.unity.postprocessing": "3.1.1"
 	},
 	"vpmDependencies": {
-		"com.vrchat.base" : "^3.2.x"
```

Comment:
I feel like there might be a situation where I want to define a more specific version of the SDK in the future if there are bigger changes to features that are touched by the toolkit, but I don't think it's important enough right now that it needs to be addressed before merging this.

Reply:
Yeah, optional dependency is in a kind of tough spot.
In my use case, I want to import VRWorldToolkit in a OSC+SteamVR Overlay project and other VCC dependencies that don't depend on the VRChat SDK.